assets/messages.yml

```diff
@@ -826,8 +952,23 @@ messages:
       name: Parity issue
       shortcut: parity
       message: |-
-        Minecraft has several different editions, and some features vary from edition to edition. Often this happens when one edition evolves faster than another, and tweaks or balances are made that don't propagate immediately to other editions.
-        Parity issues are not tracked on the bug tracker but should instead be reported on the [Feedback website|https://feedback.minecraft.net/hc/en-us/community/topics/360001191251-Parity].
+        *Thank you for your report!*
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+
+        You have posted a parity issue. Minecraft has several different editions, and some features vary from edition to edition.
+        This is especially the case with features that were added a long time ago, because one edition evolved faster than another, and tweaks or balances were made that did not propagate immediately to other editions.
+        These differences between the different versions of the game are called "Parity issues".
+
+        We only accept bug reports about a parity issue if it fulfills the following criteria:
+        * The feature affected by the parity issue is present in both Bedrock Edition and Java Edition in the latest release or development version
+        * The feature behaves differently in one edition than in the other
+        * The parity issue was introduced in Buzzy Bees (Bedrock Edition 1.14 / Java Edition 1.15) or later and was not present before
+
+        All other parity issues are not tracked on the bug tracker but should instead be reported on the [Feedback website|https://feedback.minecraft.net/hc/en-us/community/topics/360001191251-Parity].
+
+        For more information about this, please take a look at [this post|https://www.reddit.com/r/Mojira/comments/g9rjfh/from_now_on_well_accept_bug_reports_about_certain/] on our subreddit.
+
+        %quick_links%
       fillname: []
```

Comment:
That message has been changed already, is this change still required / better than the one we already have?